EIPS/eip-4361.md

```diff
@@ -188,8 +190,49 @@ resource = "- " URI
 - Wallet implementers MAY construct a custom Sign-In With Ethereum user interface by parsing the ABNF terms into data elements for use in the interface. The display rules above still apply to custom interfaces.
 
 #### Supporting internationalization (i18n)
-- After successfully parsing the message into ABNF terms, translation may happen at the UX level per human language.
+- After successfully parsing the message into ABNF terms, translation MAY happen at the UX level per human language.
+
 
+## Security Considerations
```

Comment:
This section should be under the Test Cases section (see EIP-1 for the correct layout).

Reply:
(Although should be under backwards compatibility since you've omitted test cases)

Reply:
`Security Considerations` has now been moved and is in the proper ordering as per EIP-1.